code/modules/pai/subsystem.dm

```diff
@@ -0,0 +1,124 @@
+#define SPAM_TIME 30 SECONDS
+
+SUBSYSTEM_DEF(pai)
```

Comment:
please move this back to the subsystems folder

Reply:
I find this inconsistent with the rest of the PRs intent - but ok, no problem